nl/guides/course-management/README.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Cursusbeheer
+description: "Tutorial: cursusbeheer"
+---
+
+# Cursusbeheer
+
+**Inhoudsopgave**
+[[toc]]
+
+Een cursus wordt opgebouwd als een [leerpad](#leerpad) met oefeningen die gebundeld worden in oefeningenreeksen. De opeenvolging van oefeningenreeksen impliceert een mogelijke volgorde waarin de oefeningen kunnen opgelost worden.
+
+## Beheersrechten
+
+Binnenin een cursus kunnen de volgende beheersrechten toegekend worden:
+
+ | naam                                          | icoontje                                |permissies|
+ | ----------------------------------------------|-----------------------------------------|----------|
+ | `Student`|   *geen*|                                  studenten kunnen hun gebruikersprofiel bewerken, kunnen zich registreren voor cursussen en kunnen oplossingen indienen voor oefeningen|
+|`Cursusbeheerder`| ![cursusbeheerder](../../../images/role_icons/staff.png)|cursusbeheerders krijgen alle permissies van studenten en kunnen bovendien ook cursusbeheerders aanduiden, oplossingen van studenten bekijken en de cursus zelf bewerken| 
+
+## Cursus aanmaken
+
+Voor informatie over hoe je een nieuwe cursus aanmaakt, zie deze [pagina](../new-course).
+
+## Cursuseigenschappen instellen
+Bij beide keuzes moet je kiezen welke eigenschappen je cursus heeft. Deze worden vooraf ingevuld met de waarden van de gekopieerde cursus indien deze optie gekozen werd. Volgende eigenschappen kunnen ingesteld worden.
```

Comment:
This should be about the course/edit page (which is very similar), but the screenshot doesn't match and there are no "2 options".

Edit: after reading the rest of the page, this should be merged with the "cursus bewerken" section